src/components/UserInfo.js

```diff
@@ -0,0 +1,20 @@
+import { Fragment } from "react";
+
+export default function UserInfo({
+    userName,
+    userAddress,
+    userPhone,
+    userEmail
+}) {
+
+    return(
+        <Fragment>
+            <section className="news-card-container in-column">
+                <p className="in-line">Name: {userName}</p>
```

Comment:
maybe define the style only once for all the `p` elements inside of the `news-card-container`
```
.news-card-container > p { 
}```

Reply:
This component is not yet completed, we'll fix the definitions in later iterations

Reply:
@mircealungu also a comment.